ucl-university-college-apa.csl

```diff
@@ -214,7 +214,22 @@
     </choose>
   </macro>
   <macro name="date-sort-date">
```

Comment:
I don't understand what you're doing here? Essentially you're using the same sort date for everything except a conference paper without publication info -- but what is that getting sorted by?

Reply:
Hi Adam

I'm confused as to your comment, but mostly because I don't remember touching that part at all. Our reference style is a (slightly) modified version of APA 7 and when I check that source I see the same on line 503-505.
![image](https://user-images.githubusercontent.com/84915123/188630506-f1dd0087-73ba-4c89-a055-419d8d62b735.png)

So I would love to clarify if I could, but I don't think I have made that change myself. But I am also a total novice, so please steer me in the right direction if you can!

Reply:
The issue is the content of the macro, not its existence -- it's completely different (and likely needlessly complicated for your purposes) in APA 7. I'd suggest just reverting this to your previous version, which was simply

```
  <macro name="date-sort-date">
    <date variable="issued" form="numeric"/>
  </macro>
```

If you do need something changed about the sort behavior, could you say what?

Reply:
I see what you mean now. And no, it must have been an mistake. I saw lines that appeared twice for no reason, so I think there must have been some dodgy copy+paste on my part